### DISCUSSION TITLE
全面屏手机截屏赛马娘时顶上有黑条，如何解决？

### DISCUSSION THREAD

**Discussion by sixZI** [NateScarlet/auto-derby#197] [Q&A] [answered]
![last_screenshot](https://user-images.githubusercontent.com/47919490/135553585-172deb65-9ff6-410f-a814-c1d24c3659ab.png)
屏幕已经调节为1080x1920，但是顶上黑条的存在时游戏的实际分辨率不是1080x1920。最后图像识别不准。
有什么办法能让这游戏占满手机屏幕呢

**Comment by NateScarlet** [comment, marked as answer]
应该是你手机屏幕就不是9:16 建议用电脑版 

或者你手动添加截图后裁剪
https://github.com/NateScarlet/auto-derby/blob/e9bdca21a6dc82460b3a27941943687d52b2d651/auto_derby/clients/adb.py#L91-L104

**Reply by NateScarlet** [reply]
光裁剪还不够 还得把tap和swipe的坐标都对应偏移

**Reply by sixZI** [reply]
适当增加高度，比如说调成1080X1940，再进行剪裁，能否不用偏移坐标就能解决问题？

**Reply by NateScarlet** [reply]
坐标原点是左上角

**Reply by sixZI** [reply]
好吧，看来不太好搞，感谢答疑
